app/decorators/video_decorator.rb

```diff
@@ -0,0 +1,28 @@
+class VideoDecorator
+  attr_reader :video
+
+  def initialize(video)
+    @video = video
```

Comment:
If you called `decorate` on the video directly (`video.decorate` ) in your controller above, you could avoid writing an `initialize` method and access the video using the `object` method. You could then do things like `object.large_cover_url` below.